src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -64,55 +62,26 @@
  * about these suffixes are added.
  */
 public class SkroutzGreekStemmer {
-  public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
-  private final CharArraySet stopwords;
-  private Map<String, char[]> stepZeroExceptions;
+  private CharArraySet protectedWords = CharArraySet.EMPTY_SET;
+  private CharArrayMap<char[]> stepZeroExceptions = CharArrayMap
+          .emptyMap();
 
-  public SkroutzGreekStemmer(final CharArraySet stopwords) {
-    this.stopwords = stopwords;
+  public SkroutzGreekStemmer() {
     try {
+      this.protectedWords = loadProtected("protected_words");
       this.stepZeroExceptions = loadExceptions("step_0_exceptions");
     } catch(Exception ex) {
       System.err.println("Failed to load exceptions from yaml file");
```

Comment:
I think it would be nice to have a line like the following on the logs, so that it can be searched in production machines in case of a packaging error, accidental delete from the filesystem etc.
`[WARNING ][greek-stemmer ] Failed to load exceptions from yaml file`